src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -0,0 +1,108 @@
+
+const DEPRECATION_URL = 'https://blog.aragon.one/0-5-dao-deprecation/'
+
+const DEPRECATION_TITLE =
+  'Old 0.5 Rinkeby organizations will be deprecated on Nov. 1, 2018'
+
+const OLD_DAO_URL = 'https://old-app.aragon.org/#/'
+
+const { negative, negativeText, accent, textDimmed } = theme
+
+const DeprecatedBanner = props => (
+  <ModalConsumer>
+    {({ showModal }) => <DeprecatedDao showModal={showModal} {...props} />}
+  </ModalConsumer>
+)
+
+class DeprecatedDao extends React.Component {
+  static defaultProps = {
+    lightMode: false,
+  }
+
+  handleClick = () => {
+    const { dao, showModal } = this.props
+    showModal(DeprecatedModal, { dao })
+  }
+
+  render() {
+    const { children, lightMode } = this.props
+
+    return (
+      <React.Fragment>
+        <Banner
+          text={DEPRECATION_TITLE}
+          color={lightMode ? theme.infoBackground : negative}
+          textColor={lightMode ? theme.textPrimary : negativeText}
+          button={
+            <Button
+              onClick={this.handleClick}
+              mode={lightMode ? 'strong' : 'normal'}
+              size="mini"
+            >
+              More info
+            </Button>
+          }
+        />
+        {children}
+      </React.Fragment>
+    )
+  }
+}
+
+const DeprecatedBody = ({ dao }) => (
+  <React.Fragment>
+    <TopParagraph color={textDimmed}>
+      Over the last six months, we’ve made a lot of improvements to aragonOS.
+      Unfortunately, a number of those improvements weren’t
+      backwards-upgradeable with the contracts your organizations were
+      constructed with on Rinkeby. As a result, we’ve made the tough decision to
+      deprecate all old testnet organizations. Please see{' '}
+      <StyledSafeLink href={DEPRECATION_URL} target="_blank">
+        {DEPRECATION_URL}
+      </StyledSafeLink>{' '}
+      for more details.
+    </TopParagraph>
+    <Text.Paragraph color={textDimmed}>
+      On <time dateTime="2018-11-01">Nov. 1, 2018</time>, this organization will
+      no longer be accessible on app.aragon.one. To help migrate, and in case
+      you still need access to this organization, we will continue hosting this
+      version of Aragon Core on{' '}
+      <StyledSafeLink href={`${OLD_DAO_URL}${dao}`} target="_blank">
+        {OLD_DAO_URL}
+        {dao}
+      </StyledSafeLink>{' '}
+      until the end of <time dateTime="2019-03">March, 2019</time>.
+    </Text.Paragraph>
+  </React.Fragment>
+)
+
+const DeprecatedModal = ({ onHide, dao }) => (
+  <Modal
+    title={DEPRECATION_TITLE}
+    body={<DeprecatedBody dao={dao} />}
+    onHide={onHide}
+    More={
+      <Button.Anchor mode="strong" href={DEPRECATION_URL} target="_blank">
+        Learn more
+      </Button.Anchor>
+    }
+  />
+)
+
+const TopParagraph = styled(Text.Paragraph)`
+  margin-bottom: 1em;
+`
+
+const StyledSafeLink = styled(SafeLink)`
+  text-decoration-color: ${accent};
+  color: ${accent};
+  word-break: break-all;
```

Comment:
@bpierre I think this should be `break-word`, so that the links (which aren't that long) don't get awkwardly line split.

Reply:
Agree, we should always avoid `break-all`. 👍